static/js/import_wizard2/components/mapping_sections/constant_var.tsx

```diff
@@ -14,61 +14,109 @@
  * limitations under the License.
  */
 
+import _ from "lodash";
 import React from "react";
+import { Input } from "reactstrap";
 
 import { MappingSectionProps } from "../../templates";
-import { MappedThing, MappingVal } from "../../types";
+import { Column, MappedThing, MappingType, MappingVal } from "../../types";
 
 export function ConstantVar(props: MappingSectionProps): JSX.Element {
-  let placeMapping, dateMapping, unitMapping: MappingVal;
+  let [
+    placeMapping,
+    dateMapping,
+    unitMapping,
+    statVarMapping,
+    valueMapping,
+  ]: MappingVal[] = [];
```

Comment:
curious why using this method instead of the original method to define these variables as undefined

Reply:
This ensures all the vars have the type `MappingVal`. In the previous method, the type was only assigned to the last variable; the others all had type `any`.